lib/trivial.nix

```diff
@@ -58,9 +58,7 @@ rec {
      of the next function, and the last function returns the
      final value.
   */
-  pipe = val: functions:
-    let reverseApply = x: f: f x;
-    in builtins.foldl' reverseApply val functions;
+  pipe = builtins.foldl' (x: f: f x);
```

Comment:
Why?

Reply:
eta reduction, I suppose, though it seems a bit trivial (ha)